docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -411,15 +302,110 @@ Payload:
 }
 ```
 
-## Skip organization role sync
+## Examples of setting up generic OAuth2
 
-To prevent the sync of organization roles from the OAuth provider, set `skip_org_role_sync` to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.
-This also impacts the `allow_assign_grafana_admin` setting by not syncing the Grafana admin role from the OAuth provider.
+This section includes examples of setting up generic OAuth2 integration.
 
-```ini
-[auth.generic_oauth]
-# ..
-# prevents the sync of org roles from the Oauth provider
-skip_org_role_sync = true
-``
-```
+### Set up OAuth2 with Auth0
+
+To set up generic OAuth2 authentication with Auth0, follow these steps:
+
+1. Create an application using the following parameters:
+
+   - Name: Grafana
+   - Type: Regular Web Application
+
+1. Go to the **Settings** tab and set **Allowed Callback URLs** to `https://<grafana domain>/login/generic_oauth`.
+
+1. Click **Save Changes**.
+
+1. Use the values from the **Settings** tab to configure Grafana:
```

Comment:
Where do I configure Grafana? Is this the configuration file? If so, let's mention that, and also mention the section within that file.